script.js

```diff
@@ -1,4 +1,495 @@
+    //gameMode = RESULT_MODE;
+    gameMode = RESULT_MODE;
+  }
+
+  console.log("game mode now", gameMode);
+  return gameMode;
+};
+
+// check who win or lose
+var checkWhoWinLose = function (deckValue1, deckValue2) {
+  console.log("SEND HELP 2");
+  var playerValue = addAllCards(deckValue1);
+  var computerValue = addAllCards(deckValue2);
+
+  var endResult;
+
+  if (playerValue == 21 && computerValue != 21) {
+    endResult = `Player wins! 21 means blackjack!`;
+  } else if (playerValue != 21 && computerValue == 21) {
+    endResult = `Player loses! Computer got 21 which means blackjack!`;
+  } else if (playerValue > 21 && computerValue < 21) {
+    endResult = `Player is busted! Deck exceeds 21...Player loses.`;
+  } else if (playerValue < 21 && computerValue > 21) {
+    endResult = `Computer is busted! Deck exceeds 21...so Player wins!`;
+  } else if (playerValue == 21 && computerValue == 21) {
+    endResult = `It is a tie! Both decks are Blackjack!`;
+  } else if (playerValue == computerValue) {
+    endResult = `It is a tie! the values of both decks are the same!`;
+  } else if (playerValue > 21 && computerValue > 21) {
+    endResult = `It is a tie! Both Player and Computer busted!`;
+  } else if (playerValue < computerValue) {
+    endResult = `Player lost! Player deck is ${playerValue} which is lesser than Computer's deck of ${computerValue} `;
+  } else if (playerValue > computerValue) {
+    endResult = `Player won! Player deck is ${playerValue} which is more than Computer's deck of ${computerValue} `;
+  }
+  return endResult;
+};
+
+//determine round result win lose or tie
+var checkRoundResult = function (deckValue1, deckValue2) {
+  console.log("SEND HELP 3");
+  var playerValue = addAllCards(deckValue1);
+  var computerValue = addAllCards(deckValue2);
+
+  var roundResult;
+
+  /*
+  if (
+    (playerValue == 21 && computerValue != 21) ||
+    (playerValue < 21 && computerValue > 21) ||
+    playerValue > computerValue
+  ) {
+    roundResult = WIN;
+  } else if (
+    (playerValue != 21 && computerValue == 21) ||
+    (playerValue > 21 && computerValue < 21) ||
+    playerValue < computerValue
+  ) {
+    roundResult = LOSE;
+  } else if (playerValue == computerValue) {
+    roundResult = TIE;
+  }
+  
+  */
+  if (playerValue == 21 && computerValue != 21) {
+    roundResult = WIN;
+  } else if (playerValue != 21 && computerValue == 21) {
+    roundResult = LOSE;
+  } else if (playerValue > 21 && computerValue < 21) {
+    roundResult = LOSE;
+  } else if (playerValue < 21 && computerValue > 21) {
+    roundResult = WIN;
+  } else if (playerValue == 21 && computerValue == 21) {
+    roundResult = TIE;
+  } else if (playerValue == computerValue) {
+    roundResult = TIE;
+  } else if (playerValue > 21 && computerValue > 21) {
+    roundResult = TIE;
+  } else if (playerValue < computerValue) {
+    roundResult = LOSE;
+  } else if (playerValue > computerValue) {
+    roundResult = WIN;
+  }
```

Comment:
Might be better to group all the winning conditions together, then losing conditions, then tie. So you will get a structure like:
```
if (wincondition1 || wincondition2 ) {
  roundResult = WIN;
} else if (losecondition1 || losecondition2) {
  roundResult = LOSE;
} else {
  roundResult = TIE;
}
```
which should be clearer!